apps/condo/domains/news/components/NewsForm/InputStep/InputStepSelector.tsx

```diff
@@ -225,8 +225,15 @@ export const InputStepSelector: React.FC<InputStepSelectorProps> = ({
         showArrow:false,
         infinityScroll:true,
         placeholder:CustomSelectRecipientsPlaceholder,
-        onChange:(recipients: GetNewsSharingRecipientsQuery['recipients']) => {
-            setSharingAppFormValues(prev=>({ ...prev, scope: recipients, isAllChecked: false }))
+        onChange:(recipients: Array<string>) => {
+            try {
+                const parsedRecipients = recipients.map(recipient => JSON.parse(recipient)?.value)
+                if (parsedRecipients.some(recipient => !recipient)) return
+
+                setSharingAppFormValues(prev => ({ ...prev, scope: parsedRecipients, isAllChecked: false }))
+            } catch {
+                setSharingAppFormValues(prev => ({ ...prev, scope: prev.scope, isAllChecked: false }))
```

Comment:
Is it really needed? If the try block fails, then sharingAppFormValues will still be in its old state anyway, right?